graphql_api/types/owner/owner.graphql

```diff
@@ -15,21 +15,21 @@ type Owner {
   repository(name: String!): RepositoryResult!
```

Comment:
These fields can be null now, when the user is not part of the the field will be null and will also have error in the response with the appropriate message.